metacatalog/ext/standards_export/schemas/datacite/datacite.j2

```diff
@@ -0,0 +1,83 @@
+<resource xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns="http://datacite.org/schema/kernel-4"
+    xsi:schemaLocation="http://datacite.org/schema/kernel-4 http://schema.datacite.org/meta/kernel-4.3/metadata.xsd">
+    <identifier identifierType="UUID">{{uuid}}</identifier>
```

Comment:
Data cite only allows DOI, which need to be empty here, as long as we do not have DOIs assigned.